tools/syz-reporter/reporter.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2020 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// syz-reporter creates table information from crashes.
+// Useful tool together with tools/syz-crush to collect
+// results from the reproducer runs.
+
```

Comment:
Use empty comment line here `//` and remove this empty line before `package`.
Then it will be considered "package documentation" and will be understood by lots of Go tools, e.g.:
https://pkg.go.dev/github.com/google/syzkaller/tools/syz-benchcmp